### ISSUE TITLE
Overlapping units

### ISSUE THREAD

**Issue by Dode123** [Longwelwind/swords-and-ravens#19]
Some areas have non-optimal unit placements, causing them to overlap other areas/other units/orders. See GJ/Lanni game.

**Comment by Longwelwind** [comment]
A possible solution would be to use the width of the `unitSlot` drawn in Tiled to allow units of a region to go on two lines. This would prevent units from overlapping other regions.

**Comment by gereon77** [comment]
> A possible solution would be to use the width of the `unitSlot` drawn in Tiled to allow units of a region to go on two lines. This would prevent units from overlapping other regions.

My vote of 5 for this

**Comment by Dode123** [comment]
This is a still a problem in certain areas, for example lannisport, stoney Sept.

**Comment by Dode123** [comment]
Several areas have problems with 4 units in them. I think in these small areas, you need to have the units shown on **two rows**. That's what I did on many locations on the new thronemaster map.

**Comment by Longwelwind** [comment]
Noted, I'll put this issue as `up for grab`. There are 2 steps needed to solve this:

* First, make the box have a fixed-width set by the width of the box in the map file. This means adding a new field in `FixedRegion.ts`, if I'm not wrong.
* Second, in the map (`data/westeros.json` which can be opened with Tiled), all unit boxes must be checked to make sure their width are correct. I don't think I made sure of that when I encoded the map the first time.
